lib/manager/velaci/extract.ts

```diff
@@ -0,0 +1,34 @@
+import { logger } from '../../logger';
+import { getDep } from '../dockerfile/extract';
+import type { PackageDependency, PackageFile } from '../types';
+
+export function extractPackageFile(content: string): PackageFile | null {
+  const deps: PackageDependency[] = [];
+  try {
+    const lines = content.split('\n');
+    for (let lineNumber = 0; lineNumber < lines.length; lineNumber += 1) {
+      const line = lines[lineNumber];
+      const match = /^\s* image:\s*'?"?([^\s'"]+)'?"?\s*$/.exec(line);
+      if (match) {
```

Comment:
I would prefer using `yaml-js` parser instead of regex. like `docker-compose` manager

https://github.com/renovatebot/renovate/blob/38620be343cca6814eb5996bb1c9b7da5a618db5/lib/manager/docker-compose/extract.ts#L50-L50

Reply:
Oh, good call I recall seeing that in some other managers. I can make that change. 😄

Reply:
I think this change is what you were looking for:
https://github.com/renovatebot/renovate/pull/9935/commits/615156c4ff01140b4663336d0742e10aaa6e4ac3#diff-37577bcdbb49a81ee564792d874828ca0e7104df2a86f3f7739330384c859342R9

Reply:
Believe this commit has the change you're looking for:
https://github.com/renovatebot/renovate/pull/9935/commits/615156c4ff01140b4663336d0742e10aaa6e4ac3

I also added an additional test ensuring it reads properly formatted YAML files.